src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  &__image {
+    height: 134px;
+    margin-inline: 19px;
+    padding-bottom: 40px;
+  }
+
+  &__conteiner {
+    margin-inline: 16px;
+  }
+
+  &__title {
+    margin-block: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__stars-blok {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    margin: 0;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    color: $color-secondary;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    margin: 0;
+    color: $color-secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-number {
+    margin: 0;
+    color: $color-main;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height fro elements with content inside. use padding instead